alerts/cluster-network-operator/OVNKubernetesControllerDisconnectedSouthboundDatabase.md

```diff
@@ -0,0 +1,53 @@
+# OVNKubernetesControllerDisconnectedSouthboundDatabase
+
+## Meaning
+
+The `OVNKubernetesControllerDisconnectedSouthboundDatabase` alert is triggered
+when the OVN controller is not connected to OVN southbound database
+for more than 5 minutes.
+
+## Impact
+
+Networking control plane is degraded on the node. Existing workloads
+on the node may continue to have connectivity but any networking configuration
+update will not be applied.
+
+## Diagnosis
+
+Find the ovnkube-node pod running on the affected node.
+
+```shell
+pod=$(oc get pod -n openshift-ovn-kubernetes -l app=ovnkube-node \
+  -o jsonpath={..metadata.name} --field-selector spec.nodeName=<node>)
+```
+
+Check the southbound database connection status.
+
+```shell
+oc exec -n openshift-ovn-kubernetes ${pod} -c ovn-controller -- ovn-appctl connection-status
+```
+
+Check the logs of the ovnkube-node container looking for ovn controller error logs.
+
+```shell
+oc logs -n openshift-ovn-kubernetes ${pod} -c ovnkube-node
+```
+
+Check the logs of the ovn-controller container.
+
+```shell
+oc logs -n openshift-ovn-kubernetes ${pod} -c ovn-controller
+```
+
+Using tcpdump on the affected node verify the traffic flow to southbound database.
+
+```shell
+oc debug node/<node> -- tcpdump -i <primary_interface> tcp and port 9642
+```
+
+## Mitigation
+
+Mitigation will depend on what was found in the diagnosis section.
```

Comment:
Should we add something like if the ovnkube-node pod was having ovn-controller container CLBO-ing then we should try and recreate it or something? Or maybe its too generic.. hmm just feeling a bit bad we can't have proper mitigation steps even if we know what's wrong since we don't know the "why its wrong" part...

Reply:
I don't think recreating the pod would help but I also don't have any other mitigations that I think could work

Reply:
@tssurya yeah I feel that way too, but I guess it can be difficult to think of all cases right now, but we can update this runbook in the future, when we fix related bugs (at least that thought makes me feel better :D)

Reply:
+1
I talked to @tssurya and it would be good to update the runbooks when we are addressing alerts that fired in the wild.